Build/Projects/MonoGame.Tests.definition

```diff
@@ -84,23 +84,26 @@
         <Platforms>iOS</Platforms>
     </Compile>
 
+    <Compile Include="Framework\Bounding.cs" />
     <Compile Include="Framework\ColorTest.cs" />
+	<Compile Include="Framework\CurveKeyCollectionTest.cs" />
+	<Compile Include="Framework\CurveKeyTest.cs" />
+	<Compile Include="Framework\CurveTest.cs" />
+    <Compile Include="Framework\GamePadStateTest.cs" />
+    <Compile Include="Framework\GameTest.cs" />	
+    <Compile Include="Framework\GameTest+Methods.cs" />
+    <Compile Include="Framework\GameTest+Properties.cs" />
     <Compile Include="Framework\GestureRecognizerTest.cs" />
+    <Compile Include="Framework\MathHelperTest.cs" />
     <Compile Include="Framework\MockWindow.cs" />
     <Compile Include="Framework\PackedVectorTest.cs" />
-    <Compile Include="Framework\TouchCollectionTest.cs" />
-    <Compile Include="Framework\TouchPanelTest.cs" />
-    <Compile Include="Framework\TouchLocationTest.cs" />
-    <Compile Include="Framework\Bounding.cs" />
-    <Compile Include="Framework\MathHelperTest.cs" />
     <Compile Include="Framework\PlaneTest.cs" />
     <Compile Include="Framework\RayTest.cs" />
 	<Compile Include="Framework\RectangleTest.cs" />
-    <Compile Include="Framework\GameTest+Methods.cs" />
-    <Compile Include="Framework\GameTest+Properties.cs" />
-    <Compile Include="Framework\GameTest.cs" />
     <Compile Include="Framework\TestGameBase.cs" />
-    <Compile Include="Framework\GamePadStateTest.cs" />
+    <Compile Include="Framework\TouchCollectionTest.cs" />
+    <Compile Include="Framework\TouchLocationTest.cs" />
+    <Compile Include="Framework\TouchPanelTest.cs" />
```

Comment:
Not sure what you are trying to do here with rearranging these lines in the definition file.  Try to avoid doing this in the future as it causes conflicts for others pretty easily.